remote-workstations/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -1,122 +1,117 @@
 #!/bin/bash
 
-install_driver()
+if [ $# -eq 1 ]
```

Comment:
We should use `[[ ]]` for tests, as it's safer than `[ ]` and the shebang line says this is running in Bash. There are several places where this should be changed...

(`[ ]` is for `sh` and can have different semantics)

Reply:
I will change